src/components/Preface/Preface.tsx

```diff
@@ -0,0 +1,361 @@
+import { Grid,Box, Typography,Chip,Button } from '@mui/material'
+import Image from 'next/image';
+import React,{useState,useEffect} from 'react'
+import styles from '@styles/Home.module.css';
+import { differences,process } from '@/constants';
+
+const Preface = () => {
+    const [width,setWidth] = useState(0);
+    const desktopPros={
+        display:'grid',
+        gridTemplateColumns:'repeat(4,1fr)',
+        rowGap:'20px',
+        margin:'10%'
+    };
+    const mobileProps = {
+        display:'grid',
+        gridTemplateColumns:'repeat(2,1fr)',
+        rowGap:'20px',
+        columnGap:'10px',
+    }
+    useEffect(()=>{
+        setWidth(window.outerWidth);
+    },[])
+  return (
+      <div className={styles.preface}>
+            <Grid container justifyContent={'center'} mt={5}>
+                <Grid item lg={9} xs={12}>
+                    <Typography variant='h2' className={styles.h2}  component={'h2'}>
+                        Reshape the future 
+                    </Typography>
+                    <Typography variant='h2' className={styles.h2}  component={'h2'}>
+                        With our
+                    </Typography>
+                    <Typography align='justify' className={styles.h2} variant='h2' component={'h2'}  >
+                        <span style={{color:'#EEb609'}}>Revenue </span> 
+                        <span style={{WebkitTextStroke:'0.02rem #EEB609',WebkitTextFillColor:'#ffffff'}}>Cycle Management</span>
+                    </Typography>
+                    <Grid container xs={12} lg={9} mt={2}>
+                    <Typography align='left' variant='caption' sx={{
+                        fontFamily: 'Montserrat',
+                        fontStyle: 'normal',
+                        fontWeight: '500',
+                        fontSize:width > 800 ?'25px':'13px',
+                        lineHeight:width > 800? '40px':'20px',
+                        letterSpacing: '0.01em',
+                        color: '#6C7A89',
+                    }}>
+                    Discover new heights with our IT services.
+                    Our innovative approaches for RCM, digital marketing, 
+                    and DevOps can turn your business operations into a revenue-generating powerhouse.
+                    Let us show you how to leverage these services and take your business to the next level.
+                    </Typography>
+                    </Grid>
+                    <Grid container mt={2}>
+                    <Grid item>
+                        <Button variant={'outlined'} onClick={()=>{}} sx={{width:'140px',height:'50px',border:'1px solid #000000',borderRadius:'50px',color:'#000'}}>
```

Comment:
please move common styles to theme object instead of repeating them in all the places
(i.e) border radius values can be moved to theme file